src/styles/utils/global.scss

```diff
@@ -0,0 +1,16 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: 400;
```

Comment:
The `font-style` property is incorrectly set to `400`, which is a font-weight value. It should be set to `normal` or `italic`. Consider changing it to `font-style: normal;` if the intention is to use the normal style.